### ISSUE TITLE
Add no_tax to fees?

### ISSUE THREAD

**Issue by pippinsplugins** [easydigitaldownloads/EDD-Simple-Shipping#54]
Fees didn't used to be taxed (I think): https://secure.helpscout.net/conversation/324221158/53809/?folderId=180499

**Comment by brashrebel** [comment]
<img width="620" alt="screen shot 2017-02-26 at 10 26 54 am" src="https://cloud.githubusercontent.com/assets/4993131/23341007/2620ffe4-fc0e-11e6-8205-1736afca51cc.png">

Replicated. However, the math doesn't seem to add up. If shipping was not taxed, the taxes should be:

$39.99 x 0.0775 = $3.10

If the shipping _is_ taxed then it should be:

$44.49 x 0.0775 = $3.45

But what is actually happening is:

$??? x ??? = $3.80

So it doesn't appear to be as _simple_ as the _shipping_ 😉 fee being taxed as well.

**Comment by pippinsplugins** [comment]
@cklosowski could we get this looked at further this week? We'll need to before we can release the 2.3 update.

**Comment by cklosowski** [comment]
Nothing has changed in the release about how the fees are applied:

Old Fees Addition:
```
				EDD()->fees->add_fee( array(
					'amount'      => $amount,
					'label'       => sprintf( __( '%s Shipping', 'edd-simple-shipping' ), get_the_title( $item['id'] ) ),
					'id'          => 'simple_shipping_' . $key,
					'download_id' => $item['id'],
					'price_id'	  => isset( $item['options']['price_id'] ) ? $item['options']['price_id'] : null
				) );
```

New Fees Addition:
```
				EDD()->fees->add_fee( array(
					'amount'      => $amount,
					'label'       => $fee_label,
					'id'          => 'simple_shipping_' . $key,
					'download_id' => $item['id'],
					'price_id'	  => isset( $item['options']['price_id'] ) ? $item['options']['price_id'] : null
				) );
```

**Comment by cklosowski** [comment]
Keep in mind however, these are the defaults on adding fees:
```
$defaults = array(
	'amount'      => 0,
	'label'       => '',
	'id'          => '',
	'no_tax'      => false,
	'type'        => 'fee',
	'download_id' => 0,
	'price_id'    => NULL
);

$args = wp_parse_args( $args, $defaults );

if( $args['type'] != 'fee' && $args['type'] != 'item' ) {
	$args['type'] = 'fee';
}
```

**Comment by cklosowski** [comment]
Going back to the `2.2.2` release, we have taxed shipping fees in the past:
$5 product, $2 shipping fee, 10% tax:
![screen shot 2017-03-07 at 9 56 34 am](https://cloud.githubusercontent.com/assets/1390124/23667841/6eb8670a-031c-11e7-8b68-2120be9e9dcd.png)

**Comment by cklosowski** [comment]
yes it's calculating that tax twice, but that would have to be an EDD Core issue, not in simple shipping.

As the issue asks, do we charge tax on shipping...and the answer to that has been yes, since the previous version, so i'm going to close this and log an issue on EDD core about fees with taxes enabled counting the tax twice.

**Comment by cklosowski** [comment]
Issue in EDD core #5549
